cfgov/data_research/models.py

```diff
@@ -0,0 +1,11 @@
+from django.db import models
+
+# Used for registering users for a conference
+class ConferenceRegistration(models.Model):
+    # Required entries: name, email, sessions
```

Comment:
If you set `blank=True` but don't set `null=True` as well then these will be saved in the database as the empty string. Is there a specific reason this is better than saving them as null?